drivers/sdp3x/include/sdp3x_params.h

```diff
@@ -71,12 +71,10 @@ extern "C" {
 #ifdef 	MODULE_SDP3X_IRQ
 #define SDP3X_PARAMS                { .i2c_dev       = SDP3X_PARAM_I2C_DEV,  \
                                       .i2c_addr      = SDP3X_PARAM_I2C_ADDR, \
-                                      .irq_pin       = SDP3X_PARAM_IRQ_PIN, \
-                                      .irq_device	 = true }
+                                      .irq_pin       = SDP3X_PARAM_IRQ_PIN }
 #else
 #define SDP3X_PARAMS                { .i2c_dev       = SDP3X_PARAM_I2C_DEV,  \
-                                      .i2c_addr      = SDP3X_PARAM_I2C_ADDR, \
-                                      .irq_device    = false }
+                                      .i2c_addr      = SDP3X_PARAM_I2C_ADDR }
```

Comment:
You want to put `.irq_pin = GPIO_UNDEF` here.

Reply:
Is it not automatically initialized to `GPIO_UNDEF` ?

Reply:
No, it's automatically initialized with `0`, but the value of `GPIO_UNDEF` depends on the platform.

Reply:
Oh I see. Fixed